node.js

```diff
@@ -156,12 +156,17 @@ module.exports = {
     if (!context.dangerousExtend) checkExtend(name)
     // eslint-disable-next-line security/detect-non-literal-require
     var queries = require(require.resolve(name, { paths: ['.'] }))
-    if (!Array.isArray(queries)) {
-      throw new BrowserslistError(
-        '`' + name + '` config exports not an array of queries'
-      )
+    if (queries && queries.constructor === Object) {
```

Comment:
What do we check here?

I do not know this pattern (so other developers will not understand it too)

Reply:
What if we will have

```js
if (Array.isArray(value)) {
} else if (typeof value === "object") {
}
```

Reply:
It would be enough if we will check before that value is not empty